set.mm

```diff
@@ -217781,12 +217825,13 @@ net proof size (existence part)? $)
   ${
     $d I x y $.  $d C x y $.  $d G x y $.  $d L x y $.  $d U x y $.
     $d F x y $.  $d X x y $.
-    mreclat.i $e |- I = ( toInc ` C ) $.
+    mreclatBAD.i $e |- I = ( toInc ` C ) $.
```

Comment:
You cannot make that kind of changes in the main part of set.mm.

If you have proposals for better alternatives, first put them in your mathbox where you can say in the comment there that it is a better alternative, but do not modify the main part.  **Then**, you can propose in an issue to move these to Main or to make other adjustments.

Reply:
I do not have good proposals. The original theorem is self-referenced BAD. The sole reason I renamed this is that I wanted to prove the good version but couldn’t resolve the name clash. This is just temporary before I move the good version to main. 

If renaming is an absolute no, I can revert the change and rename mine as GOOD instead.

Reply:
The database and website are not target only at contributors. Imagine a reader who runs into the page for `mreclatBAD`. That does not give a good image of the website.

Reply:
I totally agree with you. The bad news, is that, it is already there: https://us.metamath.org/mpeuni/mreclatBAD.html

I am trying to fix it but first I need to polish the fixed one in my Mathbox. 

Anyways I will revert the change and rename mine as GOOD. Does that sound okay?

Reply:
In fact, there are more than one “BAD” theorems in main…

Reply:
Given that it's just a hypothesis, I'm fine with either renaming this hypothesis BAD or the other hypothesis GOOD

Reply:
Better to relabel the version in the mathbox, and to not have "BAD" in the main part.

Reply:
https://github.com/metamath/set.mm/pull/4248/commits/ba9e5cda4ab3315bf1dac1442095020bc8fc891c